hack/version-me.sh

```diff
@@ -0,0 +1,16 @@
+# source this file:
+# use this via `source ./hack/version-me.sh` because `./hack/version-me.sh` won't make the functions available to other scripts
+
+# We extract version information from git tags
+# the implicit contract is that our git tags will be in ~semver (three-part) format and prefaced with the letter 'v'.
+# this contract is required by the goreleaser tool and used throughout Carvel suite.
+
+# git tag version extraction graciously provided by https://github.com/vmware-tanzu/carvel-imgpkg/blob/develop/hack/build-binaries.sh
+function get_latest_git_tag {
```

Comment:
i was thinking right before merging, what does it mean to get a latest tag? should we be getting a tag on the current commit?

Reply:
I agree that "latest tag" is fragile,  but the goal was to keep a consistent interface with GoReleaser:

> GoReleaser will use the latest Git tag of your repository.

https://goreleaser.com/quick-start/